exercises/practice/leap/Leap.bat

```diff
@@ -0,0 +1,10 @@
+@echo off
+setlocal enabledelayedexpansion
+
+set "year=%~1"
+set "result="
+
```

Comment:
I do not really feel strongly about it either way, but if these are practice I have been noticing some students taking some of the "scaffolding" that is sometimes delivered as "given" and "not to be touched".   And unless told explicitly otherwise, I would want the practice exercises to provide all the areas to practice, where it is not something that "Must be done and this way and even by name, generally."

Reply:
To the extent even that the "default year" is set?  What is a default year, and how did we choose that?"

Reply:
I’m not following about the default. `year` is being set to the input for each test being checked. lt’s above the line telling the student to add their code so students just need to know that variable contains the test input. Would it address some of the potential confusion for me to rename that variable as `input`?

Reply:
Hi @kotp, may I don't get what is wrong, 'year' is a parameter and %~1 means it's the first parameter, this exercise takes one parameter so we don't need to add the second parameter, and the result added to that can be updated so at the end of the program that will be printed and unit tests able to see the output. In the code part `set "result="` is not affecting program, that means, at the start we don't have a variable that results, and `set "result="` removes the variable from memory, so can't remove a variable that doesn't initialize. In the batch file, we no need to initialize variables, when you try to assign, if not has that variable, creates one of them, so open to manipulation at the center of code.